src/selectors/address-details-selector.ts

```diff
@@ -1,3 +1,12 @@
+import { createSelector } from "@reduxjs/toolkit";
 import {AppState} from "../contracts/app-state.contracts";
 
-export const addressDetailsSelector = (state: AppState) => state.addressDetails;
\ No newline at end of file
+export const selectAddressDetails = (state: AppState) => state.addressDetails;
+
+export const isAddressFormFilled = createSelector(selectAddressDetails, addressDetails =>
```

Comment:
Somewhere you use select prefix, somewhere don't
Don't you want to make it consistent everywhere?